pkg/specgen/generate/oci.go

```diff
@@ -157,8 +160,34 @@ func canMountSys(isRootless, isNewUserns bool, s *specgen.SpecGenerator) bool {
 	return true
 }
 
+func getCGroupPermissons(unmask []string) string {
+	ro := "ro"
+	rw := "rw"
+	cgroup := "/sys/fs/cgroup"
+
+	cgroupv2, _ := cgroups.IsCgroup2UnifiedMode()
+	if !cgroupv2 {
+		fmt.Println(errors.Errorf("rw access is not supported for cgroupv1 rootless containers"))
+		return ro
+	}
+
+	if unmask != nil && unmask[0] == "ALL" {
```

Comment:
Currently if ALL then "rw". I am not sure if this is desired.

Reply:
I would say yes,  And I think you should add

strings.ToUpper(unmask[0]) == "ALL"

Reply:
I don't think is needed. This condition is described in BlockAccessToKernelFilesystems (line 154). I think that documentation indicates it should be ALL, not other variations.